templates/azure/fips.html

```diff
@@ -0,0 +1,405 @@
+          </div>
+        </li>
+      </ul>
+      <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+      <a href="https://ubuntu.com/azure/contact-us" class="p-button--positive">Let's talk Pro</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2>Launch Ubuntu Pro FIPS for Azure</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-6 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro FIPS 18.04 LTS</h3>
+      <p>Ubuntu Pro FIPS 18.04 LTS includes an azure-optimized FIPS-certified 4.15 kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2028.</p>
+      <p><a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips?tab=Overview" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-6 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro FIPS 16.04 LTS</h3>
+      <p>Ubuntu 16.04 LTS includes FIPS-certified 4.4  kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2024.</p>
+      <p><a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial-fips?tab=Overview" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>Compliance made easy</h2>
+      <p><strong>Ubuntu Pro FIPS on Azure - Compliance made easy</strong></p>
+      <p>With Azure, US government agencies and their partners have been able to combine unparalleled flexibility, breakthrough innovation, world class security and compliance. However when it comes to running a secure and compliant enterprise Linux on Azure, IT departments are often left with expensive options that sacrifice innovation for compliance. Watch this webinar on demand to see how Microsoft and Canonical, the publisher of Ubuntu, have collaborated to create a new way for government agencies to consume Linux on Azure.</p>
+     <p><a href="https://www.brighttalk.com/webcast/6793/452063" class="p-button--positive"><span class="p-link--external">Watch the webinar</span></a></p>
+    </div>
+    <div class="col-6">
+      <div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;"><script class="jsBrightTALKEmbedConfig" type="application/json">{ "channelId" : 6793, "language": "en-US", "commId" : 452063, "displayMode" : "standalone", "height" : "auto" }</script><script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script></div>
```

Comment:
Maybe @anasereijo can help with this, but check how the webinars are embedded on https://ubuntu.com/managed/cassandra and https://ubuntu.com/managed/kafka, as it's similar to this and I know that design was agreed on

Reply:
If I get the thumbnail of the webinar we can do something similar.